### DISCUSSION TITLE
add demo support

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17738] [Feature requests] [answered]
Creating the ability to replay certain game logic sequences like a "demo" file, similar to some other games from the 1990s could be useful if you want to make tutorials or watch how you beat a scenario, etc. It would also allow you to view some parts of the game you fast forwarded through at normal speed similar to a screensaver

**Comment by duncanspumpkin** [comment, marked as answer]
You mean a replay? We already have replay support.

**Comment by 733737** [comment]
are you talking about parkseq files?

**Reply by duncanspumpkin** [reply]
No I'm talking about replays where it replays all your actions so that you can replay a scenario.

**Reply by 733737** [reply]
I didn't know that was supported, but if it is thanks for whomever implemented it
